### ISSUE TITLE
Expose original id in records

### ISSUE THREAD

**Issue by michielbdejong** [mozilla-services/syncto#6]
When I look directly at the FxSync server, for instance at the 'meta' collection, I see:

``` bash
$ python sync/main.py michiel+emptysync@unhosted.org $PASSWORD get_records meta
[{u'id': u'global',
  u'modified': 1439218398.55,
  u'payload': u'{"syncID":"NOuEmrZxVWxl","storageVersion":5,"declined":[],"engines":{"clients":{"version":1,"syncID":"-qRIYq3pRFaF"},"prefs":{"version":2,"syncID":"J2d8YxLBQ68M"},"passwords":{"version":1,"syncID":"y3sQX0uYGXwz"},"tabs":{"version":1,"syncID":"MGdVuWFjTRpP"},"bookmarks":{"version":2,"syncID":"OmUGbrBTvZbn"},"addons":{"version":1,"syncID":"90lUL4MPuhpx"},"forms":{"version":1,"syncID":"Q_mWdmGZtuX9"},"history":{"version":1,"syncID":"2_MOTXJfjA9Q"}}}'}]
```

Note there is one record, with fields `id`, `modified`, and `payload`, which is  JSON string.
But when I retrieve the same record from syncto by hitting http://localhost:8000/v1/buckets/syncto/collections/meta/records, I see it like this response body:

``` json
{
    "data": [
        {
            "last_modified": 1439218398550,
            "payload": "{\"syncID\":\"NOuEmrZxVWxl\",\"storageVersion\":5,\"declined\":[],\"engines\":{\"clients\":{\"version\":1,\"syncID\":\"-qRIYq3pRFaF\"},\"prefs\":{\"version\":2,\"syncID\":\"J2d8YxLBQ68M\"},\"passwords\":{\"version\":1,\"syncID\":\"y3sQX0uYGXwz\"},\"tabs\":{\"version\":1,\"syncID\":\"MGdVuWFjTRpP\"},\"bookmarks\":{\"version\":2,\"syncID\":\"OmUGbrBTvZbn\"},\"addons\":{\"version\":1,\"syncID\":\"90lUL4MPuhpx\"},\"forms\":{\"version\":1,\"syncID\":\"Q_mWdmGZtuX9\"},\"history\":{\"version\":1,\"syncID\":\"2_MOTXJfjA9Q\"}}}",
            "id": "825a1b6a-0000-4000-8000-000000000000"
        }
    ]
}
```

I can still see that there is exactly one record in the collection, and I can see its payload JSON-string and its modified time (converted from seconds float to milliseconds integer), but the `id` field has been changed from `'global'` to `'825a1b6a-0000-4000-8000-000000000000'`.

This is not a big problem in practice for now, because I know the 'meta' collection only contains the 'global' item, so I can just retrieve `data[0]` and "assume" that that is the 'global' item. But if one day there are two items in the 'meta' collection, I wouldn't know which one of them is the 'global' item I am looking for.

Assuming there is a reason to assign these different `id`s, can you maybe expose a field like `original_id`, so that I know for sure that the item I'm looking at is the one I want to have?

**Comment by Natim** [comment]
One thing I would like to do is not to change the ID, but we need to accept other ids than UUID4 in Kinto.js for that to happen.

**Comment by Natim** [comment]
This has been fixed in #8
